src/plugins/irc-events/invite.js

```diff
@@ -1,22 +1,20 @@
-var _ = require("lodash");
 var Msg = require("../../models/msg");
 
 module.exports = function(irc, network) {
 	var client = this;
 	irc.on("invite", function(data) {
-		var target = data.to;
-
-		var chan = _.find(network.channels, {name: data.channel});
+		var chan = network.getChannel(data.channel);
 		if (typeof chan === "undefined") {
 			chan = network.channels[0];
 		}
 
 		var msg = new Msg({
 			type: Msg.Type.INVITE,
-			from: data.from,
-			target: target,
-			text: data.channel,
-			invitedYou: target.toLowerCase() === irc.me.toLowerCase()
+			time: data.time,
+			from: data.nick,
+			invited: data.invited,
+			channel: data.channel,
+			invitedYou: data.invited === irc.user.nick
```

Comment:
When you added `invitedYou` originally, it was because we didn't get a hold of `irc.me` (actually, `irc` as a whole) from the templates (or the client altogether, not sure exactly anymore).
Just checking, is that still the case here?